doc/classes/Object.xml

```diff
@@ -1121,7 +1121,7 @@
 			Notification received when the object is initialized, before its script is attached. Used internally.
 		</constant>
 		<constant name="NOTIFICATION_PREDELETE" value="1">
-			Notification received when the object is about to be deleted. Can act as the deconstructor of some programming languages.
+			Notification received when the object is about to be deleted. Can act as the destructor of some programming languages.
```

Comment:
```suggestion
			Notification received when the object is about to be deleted. Can act as the "destructor" of some programming languages.
```
Optionally could be in quotes, to match an existing reference to NOTIFICATION_PREDELETE as well as some other uses of "destructor" in quotes.

Reply:
Destructor of programming languages is a bit weird though. You don't destroy languages.

Reply:
```suggestion
			Notification received when the object is about to be deleted. Can be used like the "destructor" from some programming languages.
```
Maybe something like this, then?

Reply:
Above suggestion is good

Reply:
I think destructor is fine without the quotes. destructor is not use in an ironic or alternative fashion.